frontend/app/src/form/index.tsx

```diff
@@ -16,24 +16,84 @@ import { colors } from '../colors';
 const BountyDetailsCreationData = {
```

Comment:
Do you think we can put this in a const file under the form folder

`./src/form/formConstants.ts`

Reply:
Yeah, I have already pushed.